scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -0,0 +1,241 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    haproxy:
+      public_name: admin
+    stonith:
+      mode: sbd
+      sbd:
+        nodes:
+          "@@controller1@@":
+            devices:
+            - "/dev/sda"
```

Comment:
It may be better to use some fake value here, since we have to replace it before the installation anyway...